infrastructure/init_delete.sh

```diff
@@ -89,7 +97,7 @@ else
     echo "Failed to delete kms key alias"
 fi
 # Delete KMS key
-if aws kms schedule-key-deletion --key-id ${key_id} --pending-window-in-days 7 --region "${aws_region}"; then
+if aws kms schedule-key-deletion --key-id "${key_id}" --pending-window-in-days 7 --region "${aws_region}"; then
```

Comment:
we need to remove it from init.sh it is conflicting in 2 PRs

Reply:
Removed these commands from the init_delete.sh file